types/node/index.d.ts

```diff
@@ -38,6 +38,7 @@
 //                 Jordi Oliveras Rovira <https://github.com/j-oliveras>
 //                 Thanik Bhongbhibhat <https://github.com/bhongy>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
```

Comment:
Iirc we cannot do this as it would break too many packages?

Reply:
this breaks at least several hundreds packages and I don't think the `| null` changes require this.

Reply:
@SimonSchick sorry, I was only following what the linter was telling me to do. I'll force push over this with only changes to the null values, by implementing a Nullable

Reply:
@SimonSchick do you have a suggestion where I should implement the `NullableErrno`?